src/extractor/guidance/coordinate_extractor.cpp

```diff
@@ -32,7 +32,7 @@ const constexpr double LOOKAHEAD_DISTANCE_WITHOUT_LANES = 10.0;
 // smaller widths, ranging from 2.5 to 3.25 meters. As a compromise, we use
 // the 3.25 here for our angle calculations
 const constexpr double ASSUMED_LANE_WIDTH = 3.25;
-const constexpr double FAR_LOOKAHEAD_DISTANCE = 30.0;
+const constexpr double FAR_LOOKAHEAD_DISTANCE = 20.0;
```

Comment:
Using lanes later down, this threshold could be reduced for similar effects. Otherwise we look a bit to far.